### ISSUE TITLE
Installation and import errors- Win11, ComfyUIportable,  'no module found'

### ISSUE THREAD

**Issue by alphanumeric2** [lks-ai/anynode#23]
AnyNode won't import. 

AnyNode is looking for endless missing modules in site packages
that are supposed to be in the 'python embeded' folder.  Almost all of these modules are dependencies of the **Google generative AI** requirement(overrides?), grpcio, grpcio tools, google-cloud-resource-manager, google-api-core, proto. Specific versions of them are required so they match with each other.  

It would be helpful to now the distributions people are using that work for AnyNode.

Other than a fresh install of ComfyUI is there some other solution?

**Comment by lks-ai** [comment]
Yeah, the Google Generative AI stuff in Gemini ... I'm also having problems with protobuf. What a pain.

I don't use comfy portable, but quite possibly your solution could be downgrading your google genai library to one that has more compatible requirements with what is already there.

I'm keeping this one open in case anyone knows. I will also keep this in mind to find a solution.

**Comment by alphanumeric2** [comment]
If someone has done a successful install- and could post their 'pip list' so I could see the version numbers that are working, I would be grateful.
